coalib/coala_delete_orig.py

```diff
@@ -10,7 +10,10 @@
 
 def main(section: Section=None):
     configure_logging()
+    return start_delete_orig(section)
 
+
```

Comment:
A seperate commit for what exactly? 
This change was made by coala, and belongs to this commit itself. :)